redis/src/redis/tests/redis_hash_command_test.bal

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerinax/java;
+import ballerina/io;
+
+@test:Config {
+}
+function testHDel() {
+    var result = conn->hDel("testHDelKey", ["testHDelField1", "testHDelField2", "testHDelField3"]);
+    if (result is int) {
+        test:assertEquals(result, 3);
+    } else {
+        test:assertFail(io:sprintf("Error from Connector: %s - %s", result.reason(), <string> result.detail()["message"]));
```

Comment:
Shall we stick to 120 char width for .bal files too? Also lets format the .bal file using format command or vscode format option

Reply:
Updated in 991ac23